salt/utils/__init__.py

```diff
@@ -85,6 +85,12 @@
     HAS_WIN32API = False
 
 try:
+    import salt.utils.win_functions
+    HAS_WIN32 = True
```

Comment:
Here I would recommend `HAS_WIN32 = salt.utils.win_functions.HAS_WIN32` to make sure `win_functions` actually loads the win32 libraries.